manifests/b/BornToBeRoot/NETworkManager/2022.2.22/BornToBeRoot.NETworkManager.locale.en-US.yaml

```diff
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: BornToBeRoot.NETworkManager
+PackageVersion: 2022.2.22
+PackageLocale: en-US
+Publisher: BornToBeRoot
+PublisherUrl: https://github.com/BornToBeRoot/NETworkManager
+PublisherSupportUrl: https://github.com/BornToBeRoot/NETworkManager/issues
+# PrivacyUrl: 
+Author: BornToBeRoot
+PackageName: NETworkManager
+PackageUrl: https://borntoberoot.net/NETworkManager
+License: GNU General Public License v3.0
+LicenseUrl: https://raw.githubusercontent.com/BornToBeRoot/NETworkManager/master/LICENSE
+# Copyright: 
+CopyrightUrl: https://raw.githubusercontent.com/BornToBeRoot/NETworkManager/master/LICENSE
+ShortDescription: A powerful tool for managing networks and troubleshoot network problems!
+# Description: 
+Moniker: networkmanager
+Tags:
+- dns
+- ipscanner
+- lldp
+- pingmonitor
+- portscanner
+- putty
+- remotedesktop
+- traceroute
+- wifi
+# Agreements: 
+# ReleaseNotes: 
+ReleaseNotesUrl: https://github.com/BornToBeRoot/NETworkManager/releases/tag/2022.2.22
```

Comment:
Url changed to https://github.com/BornToBeRoot/NETworkManager/releases/tag/2022.2.22.0